src/SqsPoller/SqsPollerHostedService.cs

```diff
@@ -78,6 +80,9 @@ await _consumerResolver
                         async _ => await DeleteMessage(queueUrl, message, cancellationToken),
```

Comment:
Probably here `async/await` can lead to not catching exceptions. Example: https://stackoverflow.com/questions/18697167/use-an-async-callback-with-task-continuewith